tools/generator/filters/__init__.py

```diff
@@ -32,7 +32,8 @@ def load_library_in(env):
         c,
         cxx,
         php,
-        python
+        python,
+        caml
```

Comment:
Alphabetic ordering

Reply:
It's already not in alphabetic ordering, should I sort the whole list or just insert caml after c?

Reply:
Oh no I see, common should still be first?

Reply:
Right.